knora-ontologies/knora-base.ttl

```diff
@@ -2283,7 +2284,9 @@
 
         :resourceIcon "region.gif"^^xsd:string ;
 
-        :hasPermissions "V knora-base:UnknownUser,knora-base:KnownUser|M knora-base:ProjectMember" ;
```

Comment:
This is the state I have on my test server for BEOL. What happens if someone creates a Region now? Will it have any permissions?